source/DailyOverview/DailyJS.js

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable no-undef */
-//since all backend API calls are unknown to eslint, just disabeling no-undef
```

Comment:
hopefully this passes the linting test, but if it does then this part should be ok